gninasrc/lib/user_opts.h

```diff
@@ -20,6 +20,7 @@ struct cnn_options {
     bool gradient_check;
     bool move_minimize_frame;  //recenter with every scoring evaluation
     bool fix_receptor;
+    bool flexopt; // CNN optimization of flexible residues
     bool verbose;
```

Comment:
it doesn't need to be its own option, it should just be handled correctly when optimizing with the CNN - ultimately it looks to me like the places you're using this value we don't actually want/need any special casing - e.g. you're using it to choose between setReceptor and setReceptorFlex, but those should just be one function

Reply:
I'll revert this change following the refactoring of `setReceptor` and the way receptor and ligand atoms are stored.